dotnet/Getting_Started_VS/README.md

```diff
@@ -0,0 +1,56 @@
+# README for code example Getting_Started_VS
```

Comment:
**Getting started with Visual Studio code example** as a title? They know it's a README file?

Reply:
Went with "Getting started with the AWS SDK for .NET code example"